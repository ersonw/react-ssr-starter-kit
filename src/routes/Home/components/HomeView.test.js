import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import HomeView from './HomeView'

it('renders without crashing', () => {
  const div = document.createElement('div')
  const r = ReactDOM.render(<HomeView />, div)
  console.log(r)
  let component = renderer.create(<HomeView />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()

  const wrapper = shallow(<HomeView />)

  expect(wrapper.find('h4').length).toBe(1)
})
