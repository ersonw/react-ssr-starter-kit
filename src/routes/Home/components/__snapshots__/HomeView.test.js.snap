exports[`test renders without crashing 1`] = `
<div>
  <h4>
    Welcome!
  </h4>
  <img
    alt="This is a duck, because Redux!"
    className="duck"
    src="test-file-stub" />
</div>
`;
